shortlister/model.py

```diff
@@ -93,15 +100,51 @@ def load_applicants(path: Path):
     files = path.glob("*.pdf")
     applicants = []
     for file in files:
-        name_parts = file.stem.split("_")
-        applicant = Applicant(
-            name=" ".join(name_parts[0:2]), cv=file, scores={}, notes=""
-        )
-        applicants.append(applicant)
+        try:
+            applicant = load_applicants_from_pdf(file)
+            applicants.append(applicant)
+        except ValueError:
+            print(f"Corrupt/incorrect pdf file: {file}")
+            continue
```

Comment:
The handling of this error should move to the pdf parsing in `load_applicants_from_pdf`. If the PDF can't be read, then still return an Applicant object (might only have name and path to PDF)